content/06.methods.md

```diff
@@ -70,75 +69,72 @@ Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz
-Per author guidelines, <= 0.61 was used as a correlation coefficient cutoff at sequencing depths >10 to predict mismatched samples.
-For RNA-Seq, read strandedness was determined by running the [`infer_experiment.py` script](http://rseqc.sourceforge.net/#infer-experiment-py) on the first 200k mapped reads.
+To confirm sample matches and remove mis-matched samples from the dataset, we performed NGSCheckmate [@doi:10.1093/nar/gkx193] on matched tumor/normal CRAM files. 
+Briefly, we processed CRAMs using BCFtools to filter and call 20k common single nucleotide polymorphisms (SNPs) using default parameters [@url:https://github.com/parklab/NGSCheckMate]. 
+We used the resulting VCFs to run NGSCheckmate following [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) found in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
+Per NGSCheckmate author recommendations, we used <= 0.61 as a correlation coefficient cutoff at sequencing depths > 10 to predict mis-matched samples.
+For RNA-Seq, we determined read strandedness by running the [`infer_experiment.py` script](http://rseqc.sourceforge.net/#infer-experiment-py) on the first 200k mapped reads.
 If calculated strandedness did not match strandedness information received from the sequencing center, samples were removed from analysis.
-We required at least 60% of RNA-Seq reads mapped to the human reference or samples were removed from analysis.
+We required that at least 60% of RNA-Seq reads mapped to the human reference before samples were removed from analysis.
 
 #### Germline Variant Calling
 
 ##### SNP calling for B-allele Frequency (BAF) generation
 
-Germline haplotype calls were performed following the [GATK Joint Genotyping Workflow](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145), except the workflow was run on an individual sample basis.
-This workflow was applied to the GVCF output from the alignment workflow on normal/germline samples.
+We performed germline haplotype calls using the [GATK Joint Genotyping Workflow](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145) on individual GVCFs from the normal sample alignment workflow.
 Using only SNPs, we applied the [GATK generic hard filter suggestions](https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set) to the VCF, with an additional requirement of 10 reads minimum depth per SNP.
-This filtered VCF was used as input to Control-FREEC and CNVkit (below) for generation of BAF files.
+We used the filtered VCF as input to Control-FREEC and CNVkit (below) to generate B-allele frequency (BAF) files.
 GATK v4.0.12.0 was used for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this purpose.
```

Comment:
Note to either use the `v` or not. I removed, but can be added back in to all the versions.

```suggestion
We used GATK 4.0.12.0 for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this application.
```